orchagent/bfdorch.cpp

```diff
@@ -302,6 +304,41 @@ bool BfdOrch::register_bfd_state_change_notification(void)
     return true;
 }
 
+void BfdOrch::updateNextHopId(string alias, IpAddress peer_address, sai_object_id_t next_hop_id)
+{
+    SWSS_LOG_NOTICE("BFD: update nexthop id, next hop %s on %s, next_hop_id %llu",
```

Comment:
please dont add NOTICE to all functions, especially for those which can be repetitive.